libs/application/ui-fields/src/lib/CheckboxFormField.tsx

```diff
@@ -58,9 +58,12 @@ const CheckboxFormField: FC<Props> = ({
             useDefaultValue(field, application)
           }
           error={error}
-          options={finalOptions.map(({ label, tooltip, ...o }) => ({
+          options={finalOptions.map(({ label, subLabel, tooltip, ...o }) => ({
             ...o,
             label: HtmlParser(formatText(label, application, formatMessage)),
+            ...(subLabel && {
+              subLabel: formatText(subLabel, application, formatMessage),
+            }),
```

Comment:
This is how things are done in multiple places in the application system. Strings are formatted somewhere on their way down the component tree.
I want to suggest that we change this behaviour and do all formatting needed at the highest level and pass down a string. Not in some cases a string and in other cases a MessageDescriptor. That would also make our types more easier to follow. I feel like we have too many places where types are too broad.